### PR TITLE
update: add wsl commands to set the `vm.max_map_count`

### LINKED ISSUE #5076: [DOC] Add WSL configuration for installing opensearch
**I want to:**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Summary of my request.** 

I added the WSL configuration for Windows users to the existing `install-opensearch/index/` document. This improves user experience because windows users will also be included and they can continue to the next step.
 

**What other resources are available?** 

Steps for testing:

* press "Windows + R
* type "cmd"
* type `wsl -d docker-desktop` followed by `sysctl -w vm.max_map_count=262144`
* display the newly set minimum number of memory map areas by running `cat /proc/sys/vm/max_map_count`

### PR DESCRIPTION
**Pull request by colossus06**
Added commands to set the vm.max_map_count on WSL Windows.

### Description
With this change windows users can set the minimum `vm.max_map_count` to install opensearch on WSL.

### Issues Resolved
This change resolves #5076

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _install-and-configure/install-opensearch/index.md [hunk 1]
```diff
@@ -77,6 +77,13 @@ vm.max_map_count=262144
 
 Then run `sudo sysctl -p` to reload.
 
+For Windows workloads, you can set the `vm.max_map_count` running the following commands:
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/index.md:80]: Would this make more sense on the Window's page? https://opensearch.org/docs/latest/install-and-configure/install-opensearch/windows/

Comment (colossus06) [reply _install-and-configure/install-opensearch/index.md:80]: you may be right, i will create a PR soon

**Comment by hdhalter** [comment]
Hi @Naarcha-AWS , what's the status of this? Are we waiting for a PR that will move the topic?

**Review by vagimeli — APPROVED** [review]
LGTM

**Pull request merged**